bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/handler/ShellyRelayHandler.java

```diff
@@ -336,8 +336,20 @@ public boolean updateRelays(ShellySettingsStatus status) throws ShellyApiExcepti
         boolean updated = false;
         // Check for Relay in Standard Mode
         if (profile.hasRelays && !profile.isRoller && !profile.isDimmer) {
-            logger.trace("{}: Updating {} relay(s)", thingName, profile.numRelays);
+            double voltage = -1;
+            if (status.voltage == null && profile.settings.supplyVoltage != null) {
+                // Shelly 1PM/1L (fix)
+                voltage = profile.settings.supplyVoltage == 0 ? 110.0 : 220.0;
```

Comment:
Should be 230V, or?

Reply:
that's what the API spec says "Shelly1PM only Supply voltage of the device, 0 for 110V, 1 for 220V"